pisek/jobs/parts/tools.py

```diff
@@ -100,6 +101,41 @@ def _run(self):
             raise PipelineItemFailure("Text preprocessor compilation failed.")
 
 
+class PrepareJudgeToken(TaskJob):
```

Comment:
Nit: I would call it ``PrepareTokenJudge`` to have same name as ``RunTokenJudge``.

Reply:
I was unsure if I should use the word order used by the program `judge-token` or the more grammatical order. Apparently, I chose both. Fixed.